recipes/pixy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pixy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ksamuk/pixy/archive/{{ version }}.tar.gz
+  sha256: 40cb4db24d08d86347fe2a6dd64bd2e20f8a9c9671f19202086e5c72b670a776
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+  entry_points:
+    - pixy = pixy.__main__:main
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
I think you need to put this in as a skip, limitations on the python version only works with noarch.